core/src/main/java/org/mule/util/store/PersistentObjectStorePartition.java

```diff
@@ -446,6 +448,18 @@ protected void serialize(File outputFile, StoreValue<T> storeValue) throws Objec
         }
         finally
         {
+            if (inputStream != null)
```

Comment:
I guess there could be a utility method somewhere in our code that silently closes a stream.